Compilation error when building with tracy enabled

When building a bevy project that uses bevy_mod_scripting with the tracy feature enabled, I get the following error:

```
error[E0277]: the trait bound `Cow<'_, str>: profiling::tracing::Value` is not satisfied
   --> /Users/abeltoy/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_mod_scripting_core-0.9.3/src/bindings/function/script_function.rs:114:9
    |
114 |         profiling::scope!("Dynamic Call ", self.name().clone());
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `profiling::tracing::Value` is not implemented for `Cow<'_, str>`
    |
    = help: the following other types implement trait `profiling::tracing::Value`:
              &'a T
              &'a mut T
              (dyn StdError + 'static)
              (dyn StdError + Send + 'static)
              (dyn StdError + Send + Sync + 'static)
              (dyn StdError + Sync + 'static)
              Arguments<'a>
              Box<T>
            and 35 others
    = note: required for the cast from `&Cow<'_, str>` to `&dyn profiling::tracing::Value`
    = note: this error originates in the macro `$crate::valueset` which comes from the expansion of the macro `profiling::scope` (in Nightly builds, run with -Z macro-backtrace for more info)
```

This error occurs in `DynamicScriptFunction` `call`'s method on this line:
```rust
profiling::scope!("Dynamic Call ", self.name().clone());
```

and on `DynamicScriptFunctionMut` `call`'s method on this line:
```rust
profiling::scope!("Dynamic Call Mut", self.name().clone());
```

--- a/crates/bevy_mod_scripting_core/src/bindings/function/script_function.rs
+++ b/crates/bevy_mod_scripting_core/src/bindings/function/script_function.rs
@@ -99,7 +99,7 @@ impl DynamicScriptFunction {
         args: I,
         context: FunctionCallContext,
     ) -> Result<ScriptValue, InteropError> {
-        profiling::scope!("Dynamic Call ", self.name().clone());
+        profiling::scope!("Dynamic Call ", self.name());
         let args = args.into_iter().collect::<VecDeque<_>>();
         // should we be inlining call errors into the return value?
         let return_val = (self.func)(context, args);
@@ -155,7 +155,7 @@ impl DynamicScriptFunctionMut {
         args: I,
         context: FunctionCallContext,
     ) -> Result<ScriptValue, InteropError> {
-        profiling::scope!("Dynamic Call Mut", self.name().clone());
+        profiling::scope!("Dynamic Call Mut", self.name());
         let args = args.into_iter().collect::<VecDeque<_>>();
         // should we be inlining call errors into the return value?
         let mut write = self.func.write();

--- a/crates/xtask/src/main.rs
+++ b/crates/xtask/src/main.rs
@@ -114,11 +114,12 @@ struct Features(HashSet<Feature>);
 
 impl Default for Features {
     fn default() -> Self {
-        // should be kept up to date with the default feature + lua54
+        // should be kept up to date with the default feature + lua54 on top of anything that is handy to run locally every time
         Features::new(vec![
             Feature::Lua54,
             Feature::CoreFunctions,
             Feature::BevyBindings,
+            Feature::Tracy,
         ])
     }
 }

--- a/crates/xtask/src/main.rs
+++ b/crates/xtask/src/main.rs
@@ -49,6 +49,10 @@ enum Feature {
     Rhai,
     // Rune
     // Rune,
+
+    // Profiling
+    #[strum(serialize = "profiling/profile-with-tracy")]
+    Tracy,
 }
 
 #[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, strum::EnumIter)]
@@ -97,7 +101,8 @@ impl IntoFeatureGroup for Feature {
             Feature::MluaAsync
             | Feature::MluaMacros
             | Feature::MluaSerialize
-            | Feature::UnsafeLuaModules => FeatureGroup::ForExternalCrate,
+            | Feature::UnsafeLuaModules
+            | Feature::Tracy => FeatureGroup::ForExternalCrate,
             Feature::BevyBindings | Feature::CoreFunctions => FeatureGroup::BMSFeature,
             // don't use wildcard here, we want to be explicit
         }

